docs/michel_codec.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Michel Codec
+author: Hui-An Yang
+---
+
+The purpose of the `taquito/michel-codec` package is to convert and validate Michelson expressions between JSON-based Michelson and Micheline. This package also comes with functions like `packData`, `packDataBytes`, `unpackData` and `unpackDataBytes` to serialize any value of packable type to its optimized binary representation locally and vice versa, like Michelson instructions `PACK` and `UNPACK`.
+
+## Parser class
+To use the parser class as is, we import it from the package and initialize it.
+
+```ts
+import { Parser } from '@taquito/michel-codec'
+const p = new Parser()
+```
+### Configuration
+If you'd like to customize the parser class to `expandMacros` or `expandGlobalConstant,` you can configure its `ParserOptions` through initialization.
```

Comment:
I'd recommend saying something like, 'You can pass parameters to the parser class to blablabla' instead of 'customizing' the parser class. Customizing implies something like the taquito-http-util customization. This is just extra params.

Reply:
can you elaborate the difference of extra params and customize?

Reply:
customize would be to modify the method, refer to this doc https://taquito.io/docs/next/cancel_http_requests

Reply:
update to this "You can configure the parser class by passing `ParserOptions` through initialization with `expandMacros` and `expandGlobalConstant` properties."